### ISSUE TITLE
cytoolz has no Python 3.13 wheels

### ISSUE THREAD

**Issue by ardevd** [ethereum/eth-utils#294]
### What happened?

Our Home Assistant integration relies on `eth-utils` but since Home Assistant updated to Python 3.13, the integration breaks due to `cytoolz` lacking wheels for Python 3.13. There's [an issue for it](https://github.com/pytoolz/cytoolz/issues/209) but there's been no response or activity from anyone for the last 2 months. Ive tried to email the devs too, but I'm not getting any replies there either. 

Is it possible to replace `cytoolz` with something else? `pytoolz` for example? 

### Python Version

3.13

### Operating System

linux

**Comment by kclowes** [comment]
Yeah, we can definitely figure something out. As a workaround for now, you may be able to uninstall `cytoolz` in your environment and install `toolz`, and it should just work ™️ but lmk if that doesn't work for you for whatever reason.

**Comment by ardevd** [comment]
So the problem is that Home Assistant handles this automatically when you add the integration, and it throws an error when installing `eth-utils` due to the missing wheels.

**Comment by kclowes** [comment]
Got it. It looks like there has been [some movement on adding the python 3.13 wheels just today](https://github.com/pytoolz/cytoolz/pull/216), so I'll wait a few days to see if that gets in, but if it ends up taking more than a ~week I have a PR up with a hacky workaround.

**Comment by ardevd** [comment]
Thank you so much! And yes, seems like the cytoolz maintainers suddenly sprung to life again! Cheers!
